src/main/scala/msgpack4z/MsgpackUnion.scala

```diff
@@ -243,7 +242,15 @@ object MsgpackUnion {
       case MsgType.BINARY =>
         new MsgpackBinary(unpacker.unpackBinary())
       case MsgType.EXTENDED =>
-        MsgpackExt // TODO
+        val ext = unpacker.unpackExtensionType()
+        val len = ext.getLength
+        val data = new Array[Byte](len)
+        var i = 0
+        while (i < len) {
+          data(i) = unpacker.unpackByte()
```

Comment:
I think this is incorrent. `unpackByte` only unpack integer types, can not get raw byte.

- https://github.com/msgpack/msgpack-java/blob/0.7.0-M6/msgpack-core/src/main/java/org/msgpack/core/MessageUnpacker.java#L733-L788
- https://github.com/msgpack4z/msgpack4z-java07/blob/v0.1.4/src/main/java/msgpack4z/Msgpack07Unpacker.java#L54-L57
- https://github.com/msgpack4z/msgpack4z-java06/blob/v0.1.1/src/main/java/msgpack4z/Msgpack06Unpacker.java#L73-L76
- https://github.com/msgpack/msgpack-java/blob/msgpack-0.6.12/src/main/java/org/msgpack/unpacker/MessagePackUnpacker.java#L438-L449